.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.format.enable": true,
+  "[javascript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
```

Comment:
Run TS and JS through eslint instead of prettier directly. Mostly for consistency as it's possible to define prettier rules in a way that only eslint is aware of them. Optional.